src/JSTest/JSProcess.cs

```diff
@@ -45,7 +60,7 @@ public void EnableDebugLogs(string debugFilePath)
             this.debugFilePath = debugFilePath;
         }
 
-        public bool LaunchProcess(TestProcessStartInfo startInfo, Action<object, string> processErrorReceived, Action<object> processExitReceived)
+        public bool LaunchProcess(TestProcessStartInfo startInfo, Action<object, string> processOutputRecieived, Action<object, string> processErrorReceived, Action<object> processExitReceived)

```

Comment:
>Action<object, string> processOutputRecieived, Actio [](start = 66, length = 52)

nitpick: multiple delegates ... opportunity to move to interface..?

Reply:
fixed